stats.py

```diff
@@ -415,6 +418,52 @@ def get_comments(revision: DiffDb.Revision, sessions: Sessions) -> list[dict]:
     return comments
 
 
+def get_review_groups(sessions: Sessions) -> list[dict]:
```

Comment:
We should add some logging here indicating that we have gathered the review groups.